src/BLEDevice.h

```diff
@@ -11,6 +11,13 @@
 #include "BLERemoteCharacteristic.h"
 #include "BLERemoteService.h"
 
+struct BLEAdvertisementData
+{
+  unsigned char length;
+  unsigned char type;
+  unsigned char data[BLE_ADVERTISEMENT_DATA_MAX_VALUE_LENGTH];
+};
+
```

Comment:
I like this!

What do you think about renaming it to `BLEEirData` or something else, and also using it for the scan response arguments?